src/Umbraco.Infrastructure/Models/Membership/User.cs

```diff
@@ -322,7 +322,7 @@ public string Language
         }
 
         [IgnoreDataMember]
-        internal bool DefaultToLiveEditing
+        public bool DefaultToLiveEditing
```

Comment:
Wow blast from the past, pretty sure this can be completely deleted we don't have 'live editing' since like v4 or something

Reply:
You are right.. It is always set to false.. And never read (in backend code atleast)